.github/workflows/apply_outside_tool.yml

```diff
@@ -0,0 +1,46 @@
+name: test_coverage_with_external_tool
+
+on:
+  pull_request:
+    # open will be triggered when a pull request is created.
+    # synchronize will be triggered when a pull request has new commits.
+    # closed will be triggered when a pull request is closed.
+    types: [opened, synchronize, closed]
+
+jobs:
+  setup:
+    outputs:
+      base_commit: ${{ steps.get_base_commit.outputs.base_commit }}
+    runs-on: macOS-latest
+    steps:
+    - name: Checkout tools
+      uses: actions/checkout@v2
+      with:
+        repository: "firebase/firebase-ios-sdk"
```

Comment:
### Incremental code coverage report 
New code lines here are not covered by tests, please add tests on highlighted lines.